src/Model/AbstractTranslation.php

```diff
@@ -9,30 +9,25 @@
  *
  * @package Locastic\ApiPlatformTranslationBundle\Model
  */
-class AbstractTranslation implements TranslationInterface
+abstract class AbstractTranslation implements TranslationInterface
 {
-    /**
-     * @var null|string
-     */
-    protected $locale;
+    protected ?TranslatableInterface $translatable = null;
 
-    /**
-     * @var TranslatableInterface
-     */
-    protected $translatable;
+    public function __construct(
```

Comment:
@SpartakusMd Adding this constructor will create backward compatibility issues. Can you please explain why you decided to use it?

Reply:
It shouldn't as parameters are optional, but it's convenient at creation.

Reply:
@SpartakusMd both this fields are already set on other places, so that is why I don't think they should be in the constructor

Reply:
Removed the constructor.